draft-packages/loading-placeholder/docs/LoadingPlaceholder.stories.tsx

```diff
@@ -177,35 +150,6 @@ export const DefaultMultipleCombinedBlockAndInline = () => (
 )
 DefaultMultipleCombinedBlockAndInline.storyName =
   "Default, Multiple, Combined block and inline"
-DefaultMultipleCombinedBlockAndInline.parameters = {
-  chromatic: { disable: false },
-}
-
-export const DefaultWithoutBottomMargin = () => (
```

Comment:
Any specific reason you deleted these stories? They might still be helpful for someone using this package.

Reply:
I initially removed it to reduce stories but not necessary since I can remove the chromatic parameter. I have added them back in :)

Reply:
👍